src/java/io/bazel/rulesscala/scalac/ReportableMainClass.java

```diff
@@ -11,15 +11,28 @@
 import scala.tools.nsc.MainClass;
 import scala.tools.nsc.Settings;
 import scala.tools.nsc.reporters.Reporter;
+import java.lang.AutoCloseable;
 
 public class ReportableMainClass extends MainClass {
   private Reporter reporter;
   private final CompileOptions ops;
+  private Global _compiler = null;
```

Comment:
nit: uncommon in Java style to prefix private vars with underscore - please rename to `compiler`

Reply:
done.